web/src/invite.ts

```diff
@@ -231,7 +231,7 @@ function generate_multiuse_invite(): void {
             );
             $("#invite-user-modal .dialog_submit_button").prop("disabled", false);
             $("#invite-user-modal .dialog_exit_button").prop("disabled", false);
-            scroll_util.get_scroll_element($("#invite-user-modal"))[0].scrollTop = 0;
+            scroll_util.get_scroll_element($("#dialog_error"))[0].scrollIntoView();
```

Comment:
No need to call `scroll_util.get_scroll_element` here since `$("#dialog_error")` is not a simplebar container, it will return the element itself. You can directly call `$("#dialog_error")[0].scrollIntoView()`.

Reply:
We also define `$invite_status` element so you can directly use that variable.